src/db/client.ts

```diff
@@ -0,0 +1,59 @@
+import pg, { QueryResultRow } from 'pg';
+
+class DBClient {
```

Comment:
I think we agree to move all methods to `conn.ts` file. As that file has been used in all the places.